ArduCopter/Parameters.cpp

```diff
@@ -806,7 +806,7 @@ const AP_Param::Info Copter::var_info[] = {
 
     // @Group: RSSI_
     // @Path: ../libraries/AP_RSSI/AP_RSSI.cpp
-    GOBJECT(rssi, "RSSI_",  AP_RSSI),      
```

Comment:
@rmackay9 san. Comment . I understand. I am setting up empty space deletion in eclipse. This is the effect. I will consciously divide and try to commit.